rosbag2_transport/test/rosbag2_transport/test_play.cpp

```diff
@@ -19,6 +19,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <utility>
```

Comment:
What is this needed for?

Reply:
for the call to `std::make_unique`

Reply:
I believe that should be covered in `<memory>` per https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique.

Reply:
my bad, it's needed for `std::move`. 
https://en.cppreference.com/w/cpp/utility/move

I mean, cppcheck was complaining about it in the first place ;-)